Frameworks/UIKit/UIViewController.mm

```diff
@@ -1782,19 +1789,18 @@ - (void)_loadXamlPage {
         frame = [self _modalPresentationFormSheetFrame];
     }
 
-    // Create a template UIView
-    UIView* view = [[[UIEmptyView alloc] initWithFrame:frame] autorelease];
-    [view setAutoresizingMask:UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth];
-    [self setView:view];
-
     // Activate an instance of the XAML class and its page
     Microsoft::WRL::ComPtr<IInspectable> pageObj = nullptr;
     auto xamlType = ReturnXamlType(priv->_xamlClassName);
     xamlType->ActivateInstance(pageObj.GetAddressOf());
-
-    priv->_page = XamlControls::_createRtProxy([WXCPage class], pageObj.Get());
     [_pageMappings setObject:self forKey:(id)(void*)pageObj.Get()];
 
+    // Create a template UIView
+    priv->_page = CreateRtProxy([WXCPage class], pageObj.Get());
```

Comment:
also - why doesn't CreateRtProxy autorelease? Quite strange.

---
In reply to: [87281492](https://github.com/Microsoft/WinObjC/pull/1318#discussion_r87281492) [](ancestors = 87281492,87281267)